homeassistant/components/pi_hole/__init__.py

```diff
@@ -161,6 +161,8 @@ def _async_platforms(entry: ConfigEntry) -> list[str]:
 class PiHoleEntity(CoordinatorEntity):
     """Representation of a Pi-hole entity."""
 
+    _attr_icon: str = "mdi:pi-hole"
+
```

Comment:
Should not be needed now, as the descriptions provide the default.
```suggestion
```

Reply:
No. The default only applies to sensor entities. There are also binary_sensor and switch entities that inherit from PiHoleEntity.

https://github.com/home-assistant/core/blob/f5901265dc9518dabb4800f80c7b675c34da82fb/homeassistant/components/pi_hole/binary_sensor.py#L29

Reply:
Right I see, lets keep that out of scope right now indeed 👍